library/CMakeLists.txt

```diff
@@ -684,7 +684,7 @@ if(TINYSPLINE_CXX_AVAILABLE AND TARGET_SUPPORTS_SHARED_LIBS)
 
 		# PHP
 		if(TINYSPLINE_WITH_PHP)
-			find_package(PHP)
+			find_package(PHP COMPONENTS Development)
```

Comment:
if I understood correctly, `COMPONENTS Development` means find PHP and look for Zend, right? Wouldn't:

```cmake
find_package(PHP COMPONENTS Zend)
```
be a better name for it? `Development` could be anything :)

Reply:
MUCH better for sure, yes -- fixed now!